src/main/java/mate/acadamy/internetshop/Application.java

```diff
@@ -3,9 +3,7 @@
 import java.lang.reflect.InvocationTargetException;
```

Comment:
we don't need this class anymore, let's remove it

Reply:
Done